### DISCUSSION TITLE
Can i use tabler in my commercial project

### DISCUSSION THREAD

**Discussion by m5lil** [tabler/tabler#878] [Q&A] [answered]
Hi @beeman,

Can tabler be used in my laravel project that i intend to sell on codecanyon?

**Comment by beeman** [comment, marked as answer]
> Hi @beeman,
> 
> Can tabler be used in my laravel project that i intend to sell on codecanyon?

Hey @m5lil - I'm not related to this project except some contributions early on.

But it's licensed under the MIT license so commercial usage should be no problem.
